Tasks/UseDotNetV2/package.json

```diff
@@ -30,7 +30,7 @@
     "azure-pipelines-tool-lib": "0.11.0",
     "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
     "semver": "6.3.0",
-    "typed-rest-client": "0.11.0",
+    "typed-rest-client": "1.8.0",
```

Comment:
@tejasd1990 do you recall any rest client related issue when we updated typed-rest-client in other tasks?

Reply:
@bishal-pdMSFT it was related to redirection in case of short urls in ARG task. It was by default redirecting; to avoid that a flag was introduced in a new version of the typed-rest-client